tuf/data/roles.go

```diff
@@ -96,6 +97,16 @@ func IsBaseRole(role string) bool {
 	return false
 }
 
+// IsWildDelegation determines if a role represents a valid wildcard delegation
```

Comment:
Fun name, I had to include this
![wild_delegation](https://cloud.githubusercontent.com/assets/1865981/17040737/6da874b0-4f56-11e6-9c46-18451c6f5988.jpg)